extensions/map-layers/package.json

```diff
@@ -111,7 +111,7 @@
     "plugins": [
       "@itwin"
     ],
-    "extends": "plugin:@itwin/itwinjs-recommended"
+    "extends": "plugin:@itwin/ui"
```

Comment:
Can you put this in the next version.md for people that use our configs to know what they need to update? I expect come of the pkgs in viewer components react will need to update as well.

Reply:
We've always had these two separate configs and if anyone used itwinjs-recommended for a UI package, that was already a mistake. Do you still want me to explicitly point it out what each config is for?

Reply:
ah okay, nope you're good then. resolving this comment.